conf/app.routes

```diff
@@ -4,4 +4,6 @@ GET        /estimated-tax-liability/:nino/:year/:calcType             @controlle
 
 GET        /nino-lookup/:mtdRef                                       @controllers.IncomeSourceDetailsController.getNino(mtdRef)
 
-GET        /income-sources/:mtdRef                                    @controllers.IncomeSourceDetailsController.getIncomeSourceDetails(mtdRef)
\ No newline at end of file
+GET        /income-sources/:mtdRef                                    @controllers.IncomeSourceDetailsController.getIncomeSourceDetails(mtdRef)
+
+GET        /income-tax-view-change/income-source/:incomeSourceID/report-deadlines @controllers.ReportDeadlinesController.getReportDeadlines(incomeSourceID)
```

Comment:
The `/income-tax-view-change` context route is added by `prod.routes` so this can be removed here.

At the moment the full url would be `/income-tax-view-change/income-tax-view-change/income-source/..../report-deadlines` with a repeated context route.